registration-processor/pre-processor/registration-processor-cmd-validator-stage/src/main/java/io/mosip/registration/processor/stages/config/CMDValidatorConfig.java

```diff
@@ -0,0 +1,34 @@
+package io.mosip.registration.processor.stages.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import io.mosip.registration.processor.stages.app.CMDValidationProcessor;
+import io.mosip.registration.processor.stages.cmdvalidator.CenterValidator;
+import io.mosip.registration.processor.stages.cmdvalidator.DeviceValidator;
+import io.mosip.registration.processor.stages.cmdvalidator.MachineValidator;
+
+@Configuration
+public class CMDValidatorConfig {
```

Comment:
This file and couple of more changes are already merged to the develop branch, can you rebase and raise the PR ?